datamodel/referencedata.d/shipmentinformationtypecodes.csv

```diff
@@ -0,0 +1,11 @@
+shipment_information_type_code,shipment_information_type_name
+TRD,Transport document
+BKG,Booking
+SHI,Shipping Instructions
+SRM,Shipment Release Message
+OOG,Out-of-Gauge declaration
+DGD,Dangerous Goods Declaration
+VGM,Verified Gross Mass
+ARN,Arrival Notice
+CUS,Customs Inspection
+CAS,Cargo Survey
```

Comment:
My guess is that CAS requires a link to an equipment (Container) - the structure for ShipmentEvents does not cater for this